nsv13/code/modules/overmap/planets.dm

```diff
@@ -0,0 +1,170 @@
+/*
+Planet orbit code: a primer
+
+Hey folks, Vsauce here, and today I'm gonna learn you a little physics.
+
+The acceleration of an object needed to maintain a circular path around an object (where acceleration is directed towards the point of orbit)
+can be given by a = v^2/r where v is the orbital velocity (linear velocity) and r is the orbit of radius.
+
+Now, this can be expanded upon for all kinds of fancy stuff like mass, gravity, blah blah, but since the physics system implemented in NSV13 doesn't take mass in to account, it
+greatly simplifies the math needed to be done here.
+
+Because mass isn't taken into account, and because we have to make up arbitrary values for gravitational acceleration (a, since, we're not on a 1:1 scale with meters, obviously), we need
```

Comment:
:( 
What if we took mass into account for gravitational effects? JK..jk...Unless?

Reply:
I forget what I was ranting about, but also acceleration due to gravity is independent of mass, and I think I mentioned that somewhere in the rant, 

I think that was more aimed at the physics system in general: there's no actual accounting for mass anywhere in the physics code I could find. larger ships are handled by having less thrust defined based on their mass_flag or whatever, which is kinda an abstraction, but also provides no actual way to determine what masses would make sense for planetary bodies relative to ships, not that it would matter anyways, since distances are already skewed as well. 

mass being taken into account is abstracted via the different strengths of the gravity effects the larger overmap anomalies have